torii/src/lib.rs

```diff
@@ -279,6 +303,9 @@ pub enum Error {
     #[cfg(feature = "telemetry")]
     /// Failed to get Prometheus metrics
     Prometheus(#[source] eyre::Report),
+    #[cfg(feature = "profiling")]
```

Comment:
hm, we've had issues with conditionally included fields previously

Reply:
we already have `Prometheus` included conditionally, without conditional compilation this variant would be a dead code

Reply:
I don't think the serialization concerns are relevant here: this is an error type that is not encoded or exposed in schema.